dashboard/src/components/Clarity/converter/reactWrapper.ts

```diff
@@ -0,0 +1,75 @@
+// Code from: https://stackblitz.com/edit/react-ts-wrapper-5w8nhf
+import React from "react";
+
+export function createReactComponent<BaseComponent extends HTMLElement>(elementName: string) {
+  return class ReactWrapperComponent extends React.Component<
```

Comment:
Does this (or the future version of `createReactComponent`) force us to use class components? Is there a way to do function components? Perhaps it's possible and would avoid needing to replicate the old class methods?

Reply:
it shouldn't, I saw examples of function components with the other wrapper available. I will know more when I migrate the Header component (or other) to a function component.